core/common/src/ChangedElements.ts

```diff
@@ -11,26 +11,57 @@ import { AxisAlignedBox3dProps } from "./geometry/Placement";
  * @module Entities
  */
 
-/** @internal */
+/** Type of change bitflags that show what changed on an element
+ * @public
+ */
 export enum TypeOfChange {
+  /** A property in the element changed */
   Property = 0b1,
+  /** The geometry stream of a geometric element changed */
   Geometry = 0b10,
+  /** The placement of a geometric element changed */
   Placement = 0b100,
+  /** Indirect change occurred to this element by a related instance */
   Indirect = 0b1000,
+  /** Hidden properties of the element changed */
   Hidden = 0b10000,
 }
 
-/** @internal */
+/** Changed elements found in a changeset or between a range of changesets
+ * @public
+ */
 export interface ChangedElements {
+  /** Ids of elements that changed */
   elements: Id64String[];
+  /** ECClassIds of elements that changed */
   classIds: Id64String[];
+  /** Operation codes, see [[DbOpcode]] */
   opcodes: number[];
+  /** Type of change bitflags, see [[TypeOfChange]] */
   type: number[];
+  /** Model Ids of the changed elements */
   modelIds?: Id64String[];
+  /** Properties that changed, if any, of each changed element */
   properties?: Id64String[][];
```

Comment:
Does each property have a 64-bit Id? Given the class Id and property Id I can find the changed property?

Reply:
Oh weird, that's wrong. A property is just a string (property accessor string), will fix. And yes, with the access string and ECClass Id you can query the property value.

Reply:
Pushed fix for property. Does it make sense to show a example query here? or perhaps thats best for a tutorial/sample?

Reply:
Better for tutorial/sample I think.